### DISCUSSION TITLE
16: mandatory-script-verify-flag-failed - Sending Zcash

### DISCUSSION THREAD

**Discussion by ghost** [trustwallet/wallet-core#2180] [Q&A]
I am able to successfully sign and submit many other "bitcoin-like" coins, however, I am struggling with zcash. When broadcasting the rawTx to the network, the node responds with: 16: mandatory-script-verify-flag-failed (Script evaluated without error but finished with a false/empty top stack element)". I've tried using the Canopy consensus branch but to no avail. Any ideas?

**Comment by optout21** [comment]
Sorry, I can't help, you'll need to investigate on your own.
